massa-execution-worker/src/tests/scenarios_mandatories.rs

```diff
@@ -449,6 +449,192 @@ fn test_nested_call_gas_usage() {
     );
 }
 
+/// Test the recursion depth limit in nested calls using call SC operation
+///
+/// Create a smart contract and send it in the blockclique.
```

Comment:
Looks like the doc here is from another tests?

Reply:
I reworked the comments in 8c5ff99dc5b0330439c64783f5bb4d8cd8ff14d4, let me know if it's better now!